stixcore/products/level0/quicklook.py

```diff
@@ -148,15 +148,15 @@ def from_levelb(cls, levelb):
                                      control[['num_samples', 'index']]])
 
         control['compression_scheme_counts_skm'], control['compression_scheme_counts_skm'].meta =\
-            _get_compression_scheme(packets, 'NIX00272')
+            _get_compression_scheme(packets, ['NIXD0101', 'NIXD0102', 'NIXD0103'])
```

Comment:
i would prefer the "old" approach so the code do not have to know hard coded which are the nix parametrs used for decompression. As this might be false as the actually uncompression takes the nix names from somewhere else.